src/assets/translations/en/main.json

```diff
@@ -793,7 +793,11 @@
       "description": "Bridge via Arbitrum",
       "waitWarning": "Arbitrum’s native bridge is more secure but can take up to a week to claim.",
       "waitCta": "I understand that it will take ~7 days before I can claim funds on Ethereum",
-      "claimCta": "I understand that after claiming my funds, I’ll have to send another transaction with a fee"
+      "claimCta": "I understand that after claiming my funds, I’ll have to send another transaction with a fee",
+      "success": {
+        "tradeSuccess": "Bridge initiated",
+        "tradeComplete": "Successfully initiated bridging %{symbol} to %{chainName}."
```

Comment:
suggestion: can we add withdraw terminology here? Deposit is an actual success, not a "bridge initiated"

Reply:
Done in [3d19513](https://github.com/shapeshift/web/pull/7992/commits/3d1951386bf5e0dba41a1d9082f2a0598502f0ae)